src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -11,8 +18,8 @@ const convertInclude = ({ href, url, ...rest }) => {
   return {
     type,
     model: {
-      // `url` here should be replaced with `href` once mozart routes have been created. /*TODO: Create issue for this */
-      href: url,
+      // `url || includeUrls[type]` here should be replaced with `href` once mozart routes have been created. /*TODO: Create issue for this */
```

Comment:
Should an issue be created before this gets merged?

Reply:
yes, I will write up an issue for this to be refined and blocked pending the completion of the dependent issues.

Reply:
created https://github.com/bbc/simorgh-infrastructure/issues/859 for this

Reply:
@rhenshaw56 don't think this is the right issue ^

Reply:
thanks, Andrew. It's been updated